DependencyInjection/SmartboxIntegrationFrameworkExtension.php

```diff
@@ -191,26 +195,27 @@ protected function loadQueueDrivers(ContainerBuilder $container)
         foreach ($this->config['queue_drivers'] as $driverName => $driverConfig) {
             $driverId = self::QUEUE_DRIVER_PREFIX.$driverName;
 
-            $exceptionHandlerId =  strtolower($driverConfig['exception_handler']);
+            $exceptionHandlerId = strtolower($driverConfig['exception_handler']);
 
             $type = strtolower($driverConfig['type']);
             switch ($type) {
                 case 'rabbitmq':
                 case 'activemq':
+                case 'stomp':
                     $urlEncodeDestination = ('rabbitmq' == $type);
```

Comment:
The current stomp (rabbitmq driver type) driver had setup this `$urlEncodeDestination` as `true`, and the new value of stomp twill have this `false`

Reply:
This line was removed.